base-theme/layouts/partials/file_size_formatter.html

```diff
@@ -0,0 +1,52 @@
+{{ $fileSize := int . }}
+{{/*  Cap at 999 TB  */}}
+{{ if gt $fileSize 1098412116148224 }}{{ $fileSize = 1098412116148224 }}{{ end }}
+{{ $modifiedFileSize := 0 }}
+{{ $units := slice "B" "KB" "MB" "GB" "TB" }}
```

Comment:
```suggestion
{{ $units := slice "B" "kB" "MB" "GB" "TB" }}
```

I can look more later if no one else gets to it; just gave a quick skim.

Reply:
Thanks @ChristopherChudzicki